src/__tests__/interactions-helper.js

```diff
@@ -7,9 +7,27 @@ export const ELEMENT_TYPES = types;
 
 function getWithin(canvasOrValidTestElement) {
   if (canvasOrValidTestElement.getByRole) return canvasOrValidTestElement;
-  return within(canvasOrValidTestElement);
+  const result = within(canvasOrValidTestElement);
+  if (result instanceof Error) {
```

Comment:
`within` returns an error if it fails, and doesn't throw it. This caused weird issues in some cases. Throwing the error right away shows exactly what's wrong